### ISSUE TITLE
Ctrl+click opens links twice

### ISSUE THREAD

**Issue by Pastafarianist** [Guake/guake#1502]
- **Guake Version**: 3.4.0
- Manjaro, GNOME 3.30.2

Ctrl+clicking on any link (`http[s]://...` or the ones produced by `ls --hyperlink`) opens the corresponding program twice.

**Comment by hr567** [comment]
Same as the above environment on Archlinux.
And downgrade Guake to 3.3.2 will fix the problem.

**Comment by Pastafarianist** [comment]
I confirm that [`3.3.2-2`](https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/guake&id=d966353974b5af405d772606ecc4b8081fe33e30) does not have the issue.

**Comment by evdcush** [comment]
Same issue 18.04 buntu, XFCE, 3.4.0

Additionally, links that are wrapped in **single** quotes appear to always include the quote char on the rhs of the link.

For example, if I ctrl+click the following link from Guake: 

`'https://github.com/Guake/guake/issues/1502'` 

It will open **two** new tabs, both `https://github.com/Guake/guake/issues/1502'`. However, links wrapped in double-quotes do not appear to have the same issue.

**To replicate the behavior**, simply copy-paste the single-quoted url into Guake (or echo it if that does not work) and ctrl+click the link, eg:
```bash
$ 'https://github.com/Guake/guake/issues/1502'
# Or
$ echo "'https://github.com/Guake/guake/issues/1502'"
'https://github.com/Guake/guake/issues/1502'
```

**Comment by Pastafarianist** [comment]
@evdcush I could replicate your issue, but it is probably unrelated to the one filed here. Care to open a new issue?

**Comment by gsemet** [comment]
We used to have this on wayland, i think. Maybe the temp does not work on your system.

**Comment by Pastafarianist** [comment]
What do you mean by temp not working?

**Comment by gsemet** [comment]
there is a temporisation that uses x11 clock to avoid duplicate event

**Comment by Pastafarianist** [comment]
Could you provide some more info? `google://temporisation x11 clock` doesn't seem to yield good results. How can I check whether I have this issue?

**Comment by mlouielu** [comment]
Fixed in bce60dc by removing registering twice of on_button_press event, please take test in v3.5.0 or master `HEAD`
